src/hidash.ts

```diff
@@ -0,0 +1,82 @@
+import { forEach, slice } from 'lodash';
+import fp, { isArray, some } from 'lodash/fp';
```

Comment:
Replace full library import with imported functions.

Reply:
Needs the whole import on line `41:33`